src/slurm_plugin/instance_manager.py

```diff
@@ -776,7 +776,7 @@ def _scaling_for_jobs_multi_node(
         self.unused_launched_instances |= self._launch_instances(
```

Comment:
NIT: Let's fix the comment explaining this line of code (`... preliminary scale-all nodes attempt ...` -> `... preliminary all-or-nothing nodes attempt ...`)